apps/estuary/test/integration/application_start_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Estuary.StartTest do
+  use ExUnit.Case
+  use Divo
+
+  @elsa_endpoint Application.get_env(:estuary, :elsa_endpoint)
+  @event_stream_topic Application.get_env(:estuary, :event_stream_topic)
```

Comment:
are these two lines necessary?

Reply:
I think they are helpful for readability. We will probably use them in the future when we write test for consuming events off the event stream.

Reply:
I dont think we will since these should not need to be used in the test.  I say YAGNI unless the tests fail without these lines of code.

Reply:
They are used in the first integration test to connect to Elsa

Reply:
D'oh.  I didnt notice that.  Disregard.